test/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.t.sol

```diff
@@ -764,7 +765,7 @@ contract BancorVirtualSupplyBondingCurveFundingManagerTest is ModuleTest {
                 true, true, true, true, address(bondingCurveFundingManager)
             );
             emit VirtualCollateralAmountSubtracted(
-                sellAmountMinusFee, newVirtualCollateral - sellAmountMinusFee
+                sellAmountPlusFee, newVirtualCollateral - sellAmountPlusFee
```

Comment:
In addition to the comment from above:
What is happening here that the test is still running through although changes were made here?

Reply:
Unsure what you mean. The emit as well as the asset got updated here in this test

Reply:
Ah okay I think I got it:
I didnt see that _subVirtualCollateralAmount(redeemAmount + feeAmount) would trigger an event here